internal/authtoken/repository.go

```diff
@@ -196,11 +201,13 @@ func (r *Repository) ValidateToken(ctx context.Context, id, token string, opt ..
 	// If the token is too old or stale invalidate it and return nothing.
 	exp, err := ptypes.Timestamp(retAT.GetExpirationTime().GetTimestamp())
 	if err != nil {
-		return nil, fmt.Errorf("validate token: expiration time : %w", err)
+		// hmm
+		return nil, errors.Wrap(err, op, errors.WithMsg("expiration time"))
 	}
 	lastAccessed, err := ptypes.Timestamp(retAT.GetApproximateLastAccessTime().GetTimestamp())
 	if err != nil {
-		return nil, fmt.Errorf("validate token: last accessed time : %w", err)
+		// maybe should just return "invalid token?"
+		return nil, errors.Wrap(err, op, errors.WithMsg("last accessed time"))
```

Comment:
Yeah not sure on this @jimlambrt similar to above...

Reply:
same answer...